aten/src/ATen/native/cuda/linalg/BatchLinearAlgebra.cpp

```diff
@@ -1834,17 +1839,17 @@ void geqrf_magma(const Tensor& input, const Tensor& tau) {
 }
 
 void geqrf_kernel(const Tensor& input, const Tensor& tau) {
-#ifdef CUDART_VERSION
+#if defined(CUDART_VERSION) || defined(USE_ROCM)
```

Comment:
Should we just delete this if, as you did in the other changes below?

Reply:
I believe this #ifdef is needed because in the event that we are not using CUDA or ROCM, then the logic just defaults to take the magma route on line 1874. At least that is my understanding of the code. Please correct me if I am wrong.